image/v1/types.go

```diff
@@ -331,7 +348,11 @@ type TagEventCondition struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ImageStreamMapping represents a mapping from a single tag to a container image as
-// well as the reference to the container image stream the image came from.
+// well as the reference to the container image stream the image came from. This resource
```

Comment:
how does this mapping map to an imagestream?  it only has an "image" field and a "tag" field?